.eslintrc.js

```diff
@@ -2,16 +2,56 @@
 // if you're looking for a client (especially React) or server lint rule
 // go to the configs in their respective directories
 module.exports = {
+	env: { 'jest/globals': true },
 	extends: [
 		'airbnb', // Airbnb style guide
+		'eslint:recommended',
```

Comment:
Airbnb and eslint:recommended are generally not meant to go together - I'd pick one (airbnb imo)